index.js

```diff
@@ -0,0 +1,60 @@
+const submitBtn = document.querySelector('.submit-btn');
+const bookSection = document.querySelector('.books-section');
+const bookTitle = document.querySelector('.title');
+const bookAuthor = document.querySelector('.author');
+
+let myBooks = [];
+if (localStorage.getItem('books')) {
+  const locaBooks = localStorage.getItem('books');
+  const readyBooks = JSON.parse(locaBooks);
+  myBooks = [...readyBooks];
+  for (let i = 0; i < myBooks.length; i += 1) {
+    const book = document.createElement('div');
+    book.classList.add('book');
+    book.innerHTML = `
+  <h1 class="header">${myBooks[i].title}</h1>
+  <h3 class="auther">${myBooks[i].author}</h3>
+  <button class="remove-btn">remove</button>
+  `;
+    bookSection.appendChild(book);
+  }
+
+  const removeBtn = document.querySelectorAll('.remove-btn');
+  removeBtn.forEach((btn) => {
+    btn.addEventListener('click', (event) => {
+      const header = event.target.closest('div');
+      const key = header.firstElementChild.innerHTML;
+      event.target.closest('div').remove();
+      const filtered = myBooks.filter((book) => book.title !== key);
+      localStorage.setItem('books', JSON.stringify(filtered));
+    });
+  });
+}
```

Comment:
- [ ] Kindly the delete method doesn't work correctly, sometimes doesn't delete the task, this is because you are using the title to delete, but it's better to add an id for each task to delete for this specific id instead of by title.

![image](https://user-images.githubusercontent.com/35535130/163857305-56825a4e-b734-4e1e-8f61-b633cbd873ff.png)